force-di/main/classes/di_Injector.cls

```diff
@@ -139,14 +139,14 @@ public class di_Injector {
             // TODO: Support Namespace
             for(di_Binding__mdt bindingConfig :
                 [select QualifiedAPIName, DeveloperName, NamespacePrefix,
-                        Type__c, To__c, BindingObject__c, BindingSequence__c
+                        Type__c, To__c, BindingObject__c, BindingObject__r.QualifiedApiName, BindingSequence__c
```

Comment:
@ImJohnMDaniel @afawcett Any concerns here about a null pointer exception if the binding is not related to an object?